### ISSUE TITLE
Change how protobuf is compiled

### ISSUE THREAD

**Issue by omzmarlon** [omzmarlon/ManagementApp#17]
Right now, protobuf compilation and cleaning is tied to our sbt compile, and sbt clean.
This is causing this problem:
If we clean our targets, generated protobuf files will also be cleaned.
And we can only generate new protobuf if we run sbt compile (which would compile the whole project!)

Protobuf compilation and cleaning should be separated from our project compilation

**Comment by tomyang729** [comment]
Good point. We should create a new sbt task for protobuf compilation and cleaning
